website/src/css/pages/_code-insights.scss

```diff
@@ -0,0 +1,95 @@
+.code-insights-page {
+    &__hero-and-header {
+        background-image: url('/code-insights/code-insights-illustration.svg'),
+            url('/code-insights/code-insights-page-bg.svg'),
+            // Fallback while the SVGs are loading
+            radial-gradient(ellipse at bottom right, rgba($blue-mist, 0.5), transparent),
+            radial-gradient(ellipse at bottom left, rgba($light-green2, 0.5), transparent);
+        background-repeat: no-repeat, no-repeat;
+        background-position: center right, top left;
```

Comment:
I'm seeing the hero cut off after around 2045px. I'm not sure if adding `background-size: cover;` would suffice but we should span the background across the entire width.

![Screen Shot 2022-02-18 at 1 35 06 PM](https://user-images.githubusercontent.com/1733936/154754950-669a5c05-e5a5-49e2-97a2-e8ccc1496129.png)

Reply:
100% yay!